app/routes/wow.weekly-price-group-delta.tsx

```
@@ -0,0 +1,1475 @@
+                      <option value={1500}>1500%</option>
+                      <option value={2000}>2000%</option>
+                      <option value={3000}>3000%</option>
+                      <option value={4000}>4000%</option>
+                      <option value={5000}>5000%</option>
+                    </select>
+                  </div>
+                </div>
+
+                <div className="mx-4 border-t border-gray-300 dark:border-gray-600 my-2"></div>
+
+                <h4 className="font-medium px-4 pb-2">Show/Hide Items</h4>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)